tests/purs/warning/DeprecatedFFIPrime.out

```diff
@@ -0,0 +1,56 @@
+Error 1 of 4:
+
+  at tests/purs/warning/DeprecatedFFIPrime.purs:5:1 - 10:28 (line 5, column 1 - line 10, column 28)
+
+    In the FFI module for [33mMain[0m:
+
+      The identifier [33ma'[0m contains a [33m'[0m.
```

Comment:
I think people might not realise that a `'` is the same thing as a prime, which might make this wording confusing. How about having the error say this:

```
The identifier a' contains a prime (').
Primes in identifiers ...
```

Reply:
I applied your suggestion!